kroxylicious-operator/src/test/java/io/kroxylicious/kubernetes/operator/ResourceStateTest.java

```diff
@@ -38,20 +38,26 @@ void shouldReturnConditionWithLargerGeneration() {
                 .withType(Condition.Type.ResolvedRefs)
                 .withStatus(Condition.Status.FALSE)
                 .withLastTransitionTime(Instant.EPOCH)
+                .withReason("RESOLVE_FAILURE")
```

Comment:
the reason description says reason should be camel case, I guess we should go camelcase if that's common in k8s world

Reply:
Yeah